server/bodyscan/services/bodyScanService.test.ts

```diff
@@ -1,24 +1,21 @@
 import { BodyScan } from 'body-scan'
-import { HmppsAuthClient } from '../../data'
+import { createMockHmppsAuthClient } from '../../data/__testutils/mocks'
 import { BodyScanClient } from '../data'
 import BodyScanService from './bodyScanService'
 
-jest.mock('../../data')
 jest.mock('../data')
 
 const token = 'some token'
 
 describe('Body scan service', () => {
-  let bodyScanClient: jest.Mocked<BodyScanClient>
-  let hmppsAuthClient: jest.Mocked<HmppsAuthClient>
+  const bodyScanClient = new BodyScanClient(null) as jest.Mocked<BodyScanClient>
+  const hmppsAuthClient = createMockHmppsAuthClient()
```

Comment:
I haven't changed the bodyscan service specific code yet - it's only used in two controllers so far